README.md

```diff
@@ -136,6 +135,22 @@ value should be less than ~5000, otherwise fuzzer can miss new interesting input
 due to hash collisions. And finally ```uptime``` is uptime of the process. This same
 information is also served via http (see the ```-http``` flag).
 
+## Modules support
+
+go-fuzz has preliminary support for fuzzing [Go Modules](github.com/golang/go/wiki/Modules). 
+go-fuzz respects the standard `GO111MODULE` environment variable, which can be set to `on`, `off`, or `auto`. 
+
+You must have a `require` directive for `github.com/dvyukov/go-fuzz` in your go.mod file. Two approaches are:
```

Comment:
I think we can simplify this. Instead of these three paragraphs ("You must", "1.", and "2."), how about just:

```
go-fuzz-build will add a `require` for `github.com/dvyukov/go-fuzz` to your go.mod. If desired, you may remove this once the build is complete.
```

Reply:
Done.